pkg/storage/mysql/mysql.go

```diff
@@ -33,6 +33,7 @@ type MySQL struct {
 	logger                 logger.Logger
 	maxTuplesPerWriteField int
 	maxTypesPerModelField  int
+	currentTime            storage.TimeProvider
```

Comment:
```suggestion
	timeProvider            storage.TimeProvider
```

Reply:
Obsolete as we don't have the time provider anymore